apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -280,20 +286,37 @@ const AttestationsTable = ({
                   </Tr>
                 ))}
               </Thead>
-              <Tbody {...getTableBodyProps()}>
-                {rows.map(row => {
-                  prepareRow(row);
-                  return (
-                    <Tr {...row.getRowProps()}>
-                      {row.cells.map(cell => (
-                        <Td {...cell.getCellProps()} borderColor="gray.100">
-                          {cell.render('Cell')}
-                        </Td>
-                      ))}
-                    </Tr>
-                  );
-                })}
-              </Tbody>
+              <InfiniteScroll
+                dataLength={rows.length}
+                next={() => {
+                  daoContributionPagination.next();
+                }}
+                scrollThreshold={0.8}
```

Comment:
Maybe make this threshold .9? Does it feel smoother with that value?

Reply:
It does feel smoother. Also for slower connection, this should be more proactive.